src/main/java/seedu/address/model/person/Person.java

```diff
@@ -23,17 +20,19 @@
     // Data fields
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
+    private final UniqueLogList logs = new UniqueLogList();
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags, List<Log> logs) {
```

Comment:
No real functional reason aside from ease of implementation. If down the line we will support a `findlog` feature, then maybe refactoring to a `Set` object may be wise. Otherwise I think its unnecessary complexity.